src/egon/data/datasets/industrial_gas_demand.py

```diff
@@ -609,34 +613,50 @@ def download_industrial_gas_demand():
 
     """
-    correspondance_url = (
-        "http://opendata.ffe.de:3000/region?id_region_type=eq.38"
-    )
-
-    # Read and save data
-    result_corr = requests.get(correspondance_url)
-    target_file = Path(".") / "datasets/gas_data/demand/region_corr.json"
-    os.makedirs(os.path.dirname(target_file), exist_ok=True)
-    pd.read_json(result_corr.content).to_json(target_file)
+    try:
+        correspondance_url = (
+            "http://opendata.ffe.de:3000/region?id_region_type=eq.38"
+        )
 
-    carriers = {"H2_for_industry": "2,162", "CH4_for_industry": "2,11"}
-    url = "http://opendata.ffe.de:3000/opendata?id_opendata=eq.66&&year=eq."
+        # Read and save data
+        result_corr = requests.get(correspondance_url)
+        target_file = Path(".") / "datasets/gas_data/demand/region_corr.json"
+        os.makedirs(os.path.dirname(target_file), exist_ok=True)
+        pd.read_json(result_corr.content).to_json(target_file)
 
-    for scn_name in ["eGon2035", "eGon100RE"]:
-        year = str(
-            get_sector_parameters("global", scn_name)["population_year"]
+        carriers = {"H2_for_industry": "2,162", "CH4_for_industry": "2,11"}
+        url = (
+            "http://opendata.ffe.de:3000/opendata?id_opendata=eq.66&&year=eq."
         )
 
-        for carrier, internal_id in carriers.items():
-            # Download the data
-            datafilter = "&&internal_id=eq.{" + internal_id + "}"
-            request = url + year + datafilter
-
-            # Read and save data
-            result = requests.get(request)
-            target_file = (
-                Path(".")
-                / "datasets/gas_data/demand"
-                / (carrier + "_" + scn_name + ".json")
+        for scn_name in ["eGon2035", "eGon100RE"]:
+            year = str(
+                get_sector_parameters("global", scn_name)["population_year"]
             )
-            pd.read_json(result.content).to_json(target_file)
+
+            for carrier, internal_id in carriers.items():
+                # Download the data
+                datafilter = "&&internal_id=eq.{" + internal_id + "}"
+                request = url + year + datafilter
+
+                # Read and save data
+                result = requests.get(request)
+                target_file = (
+                    Path(".")
+                    / "datasets/gas_data/demand"
+                    / (carrier + "_" + scn_name + ".json")
+                )
+                pd.read_json(result.content).to_json(target_file)
+    except:
+        logger.warning(
+            """
+        Due to temporal problems in the FFE platform, data for the scenarios
+        eGon2035 and eGon100RE are imported lately from csv files. Data for
+        other scenarios is unfortunately unavailable.
+            """
+        )
+        shutil.copytree(
+            "data_bundle_powerd_data/demand",

```

Comment:
I would probably use a more precise name of the subfolder since there are so many demands in our data. E.g. industrial_gas_demand_timeseries What do you think?

Reply:
Good point. The folder was renamed as _industrial_gas_demand_ in 8d2b897364a5d41eeba8be684cdab305b729e5a8.